xml/System.Xml/XmlException.xml

```diff
@@ -85,7 +85,31 @@
       <Parameters />
       <Docs>
         <summary>Initializes a new instance of the <see langword="XmlException" /> class.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+      ## Remarks  
+       The `XmlException` is a generic XML-related error that is thrown if problems occurr during XML processing. An example use case of the `XmlException` class:
+ 
```

Comment:
It would be preferable to create this as a compilable example in the dotnet/samples repo and referenced from here. I can be stored in samples/snippets/csharp/api/system.xml/xmlexception/. It then is referenced here using a markdown exception:

\[!code-csharp\[*label*](~/samples/snippets/csharp/api/system.xml/xmlexception/*filename*)].

Also, can you add a corresponding Visual Basic example? Its location is samples/snippets/visualbasic/api/system.xml/xmlexception/. Its language identifier is code-vb.